### PR TITLE
Glin

### PR DESCRIPTION
**Pull request by lingor93**
coding exercise for Gordan

### CODE REVIEW

File: src/test/java/com/ts/person/PersonTests.java [hunk 1]
```diff
@@ -1,24 +1,114 @@
...
+        personRepository.deleteAll();
+    }
+
     @Test
     void getAllPersons() throws Exception {
         mvc.perform(get("/persons")).andExpect(status().isOk()).andDo(print());
     }
+
+    @Test
+    void getAllPersons_shouldReturnPersonSortedByLastNameAscending() throws Exception {
```
Comment (burlesgonewild) [inline src/test/java/com/ts/person/PersonTests.java:70]: Good job writing tests to cover the new functionality!

File: src/test/resources/application.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+logging:
```
Comment (burlesgonewild) [inline src/test/resources/application.yml:1]: Do we always need a test application.yml?

File: src/main/java/com/ts/person/ErrorController.java [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+@ControllerAdvice
+public class ErrorController {
```
Comment (burlesgonewild) [inline src/main/java/com/ts/person/ErrorController.java:16]: Nice use of ControllerAdvice, this keeps the controller clean.

File: src/test/resources/application.yml (hunk 2 above)
Comment (lingor93) [reply src/test/resources/application.yml:1]: No, I added it just in case I wanted to customize some settings.

**Pull request closed without merging**